texttospeech/cloud-client/README.md

```diff
@@ -46,30 +46,8 @@ Synthesize text to an output audio file. [Java Code](https://github.com/GoogleCl
 mvn exec:java -DQuickstart
```

Comment:
Just one nit. Should we be removing this one too ?
```
mvn exec:java -DQuickstart
```

Reply:
I don't think so, cus the quick start is still in a main() method entirely.

But I did totally delete the pom.xml profile for it, so I should put that back in.
I think it's nice that a quickstart is still runnable, but @kurtisvg thoughts?

Reply:
Is there anything mention in the Cloud Docs with regards to it? If so, I would leave it in. If not, it probably doesn't matter.

Reply:
It does not. 
I'll remove it.